src/test/java/insat/company/platform/web/rest/IntegrationResourceIntTest.java

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
You need to change the test for the createIntegration function to have it verifiying it's checking the user or not